coffee/services/fixtableDefaultOptions.coffee

```diff
@@ -9,7 +9,6 @@ angular.module 'fixtable'
 		debugMode: false
 		editTemplate: 'fixtable/templates/editCell.html'
 		emptyTemplate: 'fixtable/templates/emptyMessage.html'
-		footerTemplate: 'fixtable/templates/footer.html'
```

Comment:
I don't think the footerTemplate default should be deleted.

Reply:
I'll put it back in. I had originally removed it because a footer template was showing up on tables that I didn't want there to be a footer. After playing around with the options for that table I was able to pass an empty string to the footer and it removes the template from that table